examples/misc_exporter_gltf.html

```diff
@@ -112,6 +117,22 @@
 
 			} );
 
+			// ---------------------------------------------------------------------
+			// Force never empty inputs for texture max width/height
+			// ---------------------------------------------------------------------
+
+			document.getElementById('option_maxwidth').addEventListener('change', function (e) {
+    			if (e.target.value == '') {
+      				e.target.value = parseInt(document.getElementById('option_maxwidth').getAttribute('max'), 10)
+    			}
+			  } );
+			  
+			document.getElementById('option_maxheight').addEventListener('change', function (e) {
+				if (e.target.value == '') {
+					e.target.value = parseInt(document.getElementById('option_maxheight').getAttribute('max'), 10)
+				}
+			} );
```

Comment:
I don't think we need the event handlers for validation here — use `inputEl.value || Infinity`, if they're empty they can be ignored.